### DISCUSSION TITLE
What about a --force flag option to "mob done" so it commits and push to our own branch in one shot.

### DISCUSSION THREAD

**Discussion by fabrobles92** [remotemobprogramming/mob#420] [Ideas]
After finishing a mob session we usually have to do the following steps:
mob done
git commit
git push

Maybe add a --force flag after merging to our branch that also git commit and git pushes to our repo all in one shot.

Final command: mob done --force and this does under the hood the following flow: mob done -> git commit -> git push 

Let me know is this is accepted so I can open the issue :D

**Comment by hollesse** [comment]
Hi @fabrobles92 this feature stands in conflict with our design rules. 

> mob owns its wip branches. It will create wip branches, make commits, push them, but also delete them.
mob requires the user to do changes in non-wip branches.

we think these design rules are important that we do not break things in production as many teams use mob directly on main therefore we do not want to make any commit to main. We just have one feature that violates this rule and this is mob start --create which pushes the local branch to origin and which should be enhanced to even create new branches locally and then pushing them. But here the risk to break something is much lower. Therefore i would reject that feature request. 

what we could do is provide a alias for this which we can place in our README. 

If you want to challenge my answer feel free ;) if not just close this discussion.

**Reply by fabrobles92** [reply]
@hollesse Thanks for the response :D. I see. And total understand the risk. However, thinking it is optional and maybe prompts for confirmation could be a way to show awareness about you are gonna do. I don't want to go against the design rules for sure tho. If that is not an option then the alias sounds like a great way to do it as well.
